development/cpp/configuring_an_ide/visual_studio_code.rst

```diff
@@ -30,20 +30,36 @@ Importing the project
 
 - Within the ``tasks.json`` file find the ``"tasks"`` array and add a new section to it:
 
-.. code-block:: js
-
-  {
-    "label": "build",
-    "type": "shell",
-    "command": "scons",
-    "group": "build",
-    "args": [
-      "platform=linuxbsd", // Change to your current platform
-      "target=debug",
-      "-j4"
-    ],
-    "problemMatcher": "$msCompile"
-  }
+.. tabs::
+  .. code-tab:: js LinuxBSD
+
+    {
+      "label": "build",
+      "group": "build",
+      "type": "shell",
+      "command": "scons",
+      "args": [
+        "platform=linuxbsd",
+        "target=debug",
+        "-j4" // Adjust the number of threads to use according to your CPU core count.
```

Comment:
We could use `-j $(nproc)` here, but I don't know if environment variables are expanded.